opentracing-api/src/main/java/io/opentracing/Span.java

```diff
@@ -19,60 +19,53 @@
  * <p>Spans are created by the {@link Tracer#buildSpan} interface.
  */
 public interface Span extends AutoCloseable {
+    /**
+     * Retrieve the associated SpanContext.
+     *
+     * @return the SpanContext that encapsulates Span state that should propagate across process boundaries.
+     */
+    SpanContext getContext();
```

Comment:
going without a getX is not abnormal when there is no setter. A lot of
libraries follow this, including zipkin. I prefer is as context()